### ISSUE TITLE
Feature Request: Media Actions

### ISSUE THREAD

**Issue by nanderer** [termux/termux-api#461]
Hi,

i like to control my phone using termux-api over ssh, but there is one thing i am missing, media control actions like play/pause, next and previous 

Here is a screenshot of the actions from the Keymapper App, which is also open source, where you sure can copy the code for the actions:

![Screenshot_20211110-005922](https://user-images.githubusercontent.com/45161625/141028215-d5e16c72-6365-4e6a-aa37-b99bfa7167f0.png)

you can find the app here:
https://f-droid.org/en/packages/io.github.sds100.keymapper/

there are sure other useful actions to implement like switching Bluetooth on/off, auto rotate on/off or do not disturb on/off

maybe the command can be named termux-action or so...

thanks implementing this actions in advance

**Comment by nanderer** [comment]
i guess it could be this code: https://github.com/sds100/KeyMapper/blob/08a52ef68d6ff319c567f5431f6aa69601e0e155/app/src/main/java/io/github/sds100/keymapper/system/media/AndroidMediaAdapter.kt

but i cant make really any sense out of it, i know a bit java, but all that stuff android and Kotlin stuff is above me

**Comment by chapmanjacobd** [comment]
I think this might be a duplicate of or dependent on #61

**Comment by Benjamin-Loison** [comment]
@chapmanjacobd As far as I understand it is not a duplicate or dependent on #61, here @nanderer seems to want to give orders to ongoing media, while in #61 it is the contrary, the purpose is to receive such media orders.

**Comment by Benjamin-Loison** [comment]
#847 meets your needs @nanderer, see *media* occurrences on https://developer.android.com/reference/android/view/KeyEvent.
